frappe/public/js/frappe/utils/utils.js

```diff
@@ -149,6 +149,11 @@ Object.assign(frappe.utils, {
 		}
 
 	},
+	get_scroll_position: function(element, additional_offset) {
+		let header_offset = $(".navbar").height() + $(".page-head").height();
+		let scroll_top = $(element).offset().top - header_offset - cint(additional_offset);
+		return scroll_top;
+	},
```

Comment:
Not used. Remove it?

Reply:
Should we leave it? Could be a useful function

Reply:
Add only when required.